resources/qml/ToggleButton.qml

```diff
@@ -12,6 +12,58 @@ Switch {
 
     implicitWidth: indicatorItem.width
 
+    state: checked ? "on" : "off"
```

Comment:
You can probably simplify this a bit by using "when" and a default state: https://doc.qt.io/qt-5/qtquick-statesanimations-states.html#the-when-property

Reply:
Huh, TIL

Reply:
I'd like to not do this, actually, because then it animates the initial setup if the switch starts in the non-default state

Reply:
That seems to still happen :3

Reply:
It turns out that I amended the "when" bit into my first commit, so I'm going to have to rewrite the original state to demonstrate what I mean.

Reply:
No, I believe you. Just that the Room Settings have the same bug with the current code :D

Reply:
Fixed